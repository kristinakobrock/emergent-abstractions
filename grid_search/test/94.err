  0%|                                                                                                     | 0/50 [00:00<?, ?it/s]  8%|#######4                                                                                     | 4/50 [00:00<00:01, 29.72it/s] 14%|#############                                                                                | 7/50 [00:00<00:01, 27.42it/s] 20%|##################4                                                                         | 10/50 [00:00<00:01, 26.63it/s] 26%|#######################9                                                                    | 13/50 [00:00<00:01, 26.25it/s] 32%|#############################4                                                              | 16/50 [00:00<00:01, 27.30it/s] 38%|##################################9                                                         | 19/50 [00:00<00:01, 27.88it/s] 44%|########################################4                                                   | 22/50 [00:00<00:01, 25.76it/s] 50%|##############################################                                              | 25/50 [00:00<00:01, 23.43it/s] 58%|#####################################################3                                      | 29/50 [00:01<00:00, 26.49it/s] 64%|##########################################################8                                 | 32/50 [00:01<00:00, 27.26it/s] 70%|################################################################3                           | 35/50 [00:01<00:00, 25.64it/s] 76%|#####################################################################9                      | 38/50 [00:01<00:00, 26.64it/s] 82%|###########################################################################4                | 41/50 [00:01<00:00, 26.27it/s] 88%|################################################################################9           | 44/50 [00:01<00:00, 26.01it/s] 94%|######################################################################################4     | 47/50 [00:01<00:00, 24.77it/s]100%|############################################################################################| 50/50 [00:01<00:00, 24.12it/s]100%|############################################################################################| 50/50 [00:01<00:00, 25.80it/s]
  0%|                                                                                                     | 0/13 [00:00<?, ?it/s] 23%|#####################4                                                                       | 3/13 [00:00<00:00, 25.42it/s] 46%|##########################################9                                                  | 6/13 [00:00<00:00, 25.57it/s] 77%|######################################################################7                     | 10/13 [00:00<00:00, 29.38it/s]100%|############################################################################################| 13/13 [00:00<00:00, 24.89it/s]100%|############################################################################################| 13/13 [00:00<00:00, 25.70it/s]
