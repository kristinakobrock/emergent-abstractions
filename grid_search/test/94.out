# ["--attributes=5", "--values=16", "--game_size=10", "--vocab_size_factor=3", "--n_epochs=60", "--batch_size=128", "--learning_rate=0.001", "--hidden_size=256", "--temp_update=0.97", "--temperature=2"]
Creating train_ds and val_ds...

Creating test_ds...
{"loss": 0.819645881652832, "acc": 0.5256048440933228, "length": 3.4625303745269775, "mode": "train", "epoch": 1}
{"loss": 0.6861723065376282, "acc": 0.62890625, "length": 1.0, "mode": "test", "epoch": 1}
{"loss": 0.7115911841392517, "acc": 0.613926112651825, "length": 2.4615676403045654, "mode": "train", "epoch": 2}
{"loss": 0.7029910087585449, "acc": 0.46171873807907104, "length": 1.0, "mode": "test", "epoch": 2}
{"loss": 0.6766756772994995, "acc": 0.5849628448486328, "length": 1.2406706809997559, "mode": "train", "epoch": 3}
{"loss": 0.6865128874778748, "acc": 0.62890625, "length": 1.0, "mode": "test", "epoch": 3}
{"loss": 0.6670781373977661, "acc": 0.6291273832321167, "length": 1.0870252847671509, "mode": "train", "epoch": 4}
{"loss": 0.6722049713134766, "acc": 0.6166015267372131, "length": 1.0, "mode": "test", "epoch": 4}
{"loss": 0.6621349453926086, "acc": 0.6261504292488098, "length": 1.0737097263336182, "mode": "train", "epoch": 5}
{"loss": 0.6683911085128784, "acc": 0.62890625, "length": 1.0, "mode": "test", "epoch": 5}
{"loss": 0.6611457467079163, "acc": 0.6293245553970337, "length": 1.0635406970977783, "mode": "train", "epoch": 6}
{"loss": 0.6631960868835449, "acc": 0.6291016936302185, "length": 1.0, "mode": "test", "epoch": 6}
{"loss": 0.6599134206771851, "acc": 0.6294821500778198, "length": 1.0511207580566406, "mode": "train", "epoch": 7}
{"loss": 0.6623224020004272, "acc": 0.62890625, "length": 1.0, "mode": "test", "epoch": 7}
{"loss": 0.6583779454231262, "acc": 0.6292978525161743, "length": 1.0458072423934937, "mode": "train", "epoch": 8}
{"loss": 0.661234974861145, "acc": 0.6292968392372131, "length": 1.0, "mode": "test", "epoch": 8}
{"loss": 0.6586669087409973, "acc": 0.6293550133705139, "length": 1.0409159660339355, "mode": "train", "epoch": 9}
{"loss": 0.660902202129364, "acc": 0.62890625, "length": 1.0, "mode": "test", "epoch": 9}
{"loss": 0.6585401296615601, "acc": 0.6293633580207825, "length": 1.0327032804489136, "mode": "train", "epoch": 10}
{"loss": 0.6617162227630615, "acc": 0.62890625, "length": 1.0, "mode": "test", "epoch": 10}
{"loss": 0.6579993367195129, "acc": 0.6293646693229675, "length": 1.0272506475448608, "mode": "train", "epoch": 11}
{"loss": 0.6609921455383301, "acc": 0.62890625, "length": 1.0, "mode": "test", "epoch": 11}
{"loss": 0.6582039594650269, "acc": 0.6293848156929016, "length": 1.0245929956436157, "mode": "train", "epoch": 12}
{"loss": 0.661083459854126, "acc": 0.62890625, "length": 1.0, "mode": "test", "epoch": 12}
{"loss": 0.6578009724617004, "acc": 0.6293684244155884, "length": 1.0180867910385132, "mode": "train", "epoch": 13}
{"loss": 0.6605931520462036, "acc": 0.62890625, "length": 1.0, "mode": "test", "epoch": 13}
{"loss": 0.6570563316345215, "acc": 0.6293783783912659, "length": 1.015212059020996, "mode": "train", "epoch": 14}
{"loss": 0.660995602607727, "acc": 0.62890625, "length": 1.0, "mode": "test", "epoch": 14}
{"loss": 0.656829297542572, "acc": 0.6293807625770569, "length": 1.0118381977081299, "mode": "train", "epoch": 15}
{"loss": 0.6606637239456177, "acc": 0.62890625, "length": 1.0, "mode": "test", "epoch": 15}
{"loss": 0.6568283438682556, "acc": 0.6293883919715881, "length": 1.0095350742340088, "mode": "train", "epoch": 16}
{"loss": 0.660284161567688, "acc": 0.62890625, "length": 1.0, "mode": "test", "epoch": 16}
{"loss": 0.6563071608543396, "acc": 0.6293204426765442, "length": 1.0077576637268066, "mode": "train", "epoch": 17}
{"loss": 0.6597572565078735, "acc": 0.62890625, "length": 1.0, "mode": "test", "epoch": 17}
{"loss": 0.6574798822402954, "acc": 0.6294376850128174, "length": 1.0064562559127808, "mode": "train", "epoch": 18}
{"loss": 0.6605761051177979, "acc": 0.6294922232627869, "length": 1.0, "mode": "test", "epoch": 18}
{"loss": 0.6575888395309448, "acc": 0.629325807094574, "length": 1.006518840789795, "mode": "train", "epoch": 19}
{"loss": 0.6622268557548523, "acc": 0.62890625, "length": 1.0, "mode": "test", "epoch": 19}
{"loss": 0.6565461158752441, "acc": 0.6293770670890808, "length": 1.0048041343688965, "mode": "train", "epoch": 20}
{"loss": 0.6617588996887207, "acc": 0.62890625, "length": 1.0, "mode": "test", "epoch": 20}
{"loss": 0.6567724943161011, "acc": 0.6293822526931763, "length": 1.0036590099334717, "mode": "train", "epoch": 21}
{"loss": 0.6604790687561035, "acc": 0.62890625, "length": 1.0, "mode": "test", "epoch": 21}
{"loss": 0.6566876173019409, "acc": 0.6293874382972717, "length": 1.003730297088623, "mode": "train", "epoch": 22}
{"loss": 0.6607637405395508, "acc": 0.62890625, "length": 1.0, "mode": "test", "epoch": 22}
{"loss": 0.6565411686897278, "acc": 0.6293899416923523, "length": 1.0023213624954224, "mode": "train", "epoch": 23}
{"loss": 0.6599917411804199, "acc": 0.62890625, "length": 1.0, "mode": "test", "epoch": 23}
{"loss": 0.6568430066108704, "acc": 0.6294492483139038, "length": 1.0017622709274292, "mode": "train", "epoch": 24}
{"loss": 0.6607352495193481, "acc": 0.62890625, "length": 1.0, "mode": "test", "epoch": 24}
{"loss": 0.6562619209289551, "acc": 0.6293884515762329, "length": 1.0014232397079468, "mode": "train", "epoch": 25}
{"loss": 0.6612330675125122, "acc": 0.62890625, "length": 1.0, "mode": "test", "epoch": 25}
{"loss": 0.656218409538269, "acc": 0.6293895244598389, "length": 1.0007776021957397, "mode": "train", "epoch": 26}
{"loss": 0.6630458831787109, "acc": 0.62890625, "length": 1.0, "mode": "test", "epoch": 26}
{"loss": 0.6557068824768066, "acc": 0.6293894052505493, "length": 1.0009772777557373, "mode": "train", "epoch": 27}
{"loss": 0.6614341139793396, "acc": 0.62890625, "length": 1.0, "mode": "test", "epoch": 27}
{"loss": 0.6562720537185669, "acc": 0.6293320655822754, "length": 1.0005053281784058, "mode": "train", "epoch": 28}
{"loss": 0.6602324843406677, "acc": 0.62890625, "length": 1.0, "mode": "test", "epoch": 28}
{"loss": 0.6565713286399841, "acc": 0.6292153000831604, "length": 1.0004663467407227, "mode": "train", "epoch": 29}
{"loss": 0.6604851484298706, "acc": 0.6294922232627869, "length": 1.0, "mode": "test", "epoch": 29}
{"loss": 0.6569257974624634, "acc": 0.6289225220680237, "length": 1.0004029273986816, "mode": "train", "epoch": 30}
{"loss": 0.6615633964538574, "acc": 0.62890625, "length": 1.0, "mode": "test", "epoch": 30}
{"loss": 0.6557937860488892, "acc": 0.6292150616645813, "length": 1.0002529621124268, "mode": "train", "epoch": 31}
{"loss": 0.6628299951553345, "acc": 0.62890625, "length": 1.0, "mode": "test", "epoch": 31}
{"loss": 0.6553870439529419, "acc": 0.6290400624275208, "length": 1.0003596544265747, "mode": "train", "epoch": 32}
{"loss": 0.6613197326660156, "acc": 0.62890625, "length": 1.0, "mode": "test", "epoch": 32}
{"loss": 0.655701756477356, "acc": 0.6290395855903625, "length": 1.0001775026321411, "mode": "train", "epoch": 33}
{"loss": 0.6606501340866089, "acc": 0.6287109851837158, "length": 1.0, "mode": "test", "epoch": 33}
{"loss": 0.6553159356117249, "acc": 0.6292144060134888, "length": 1.0004619359970093, "mode": "train", "epoch": 34}
{"loss": 0.6603012681007385, "acc": 0.6291016936302185, "length": 1.0, "mode": "test", "epoch": 34}
{"loss": 0.6558064222335815, "acc": 0.6293319463729858, "length": 1.000562071800232, "mode": "train", "epoch": 35}
{"loss": 0.6612544655799866, "acc": 0.62890625, "length": 1.0, "mode": "test", "epoch": 35}
{"loss": 0.6553922891616821, "acc": 0.6293320059776306, "length": 1.00032377243042, "mode": "train", "epoch": 36}
{"loss": 0.660650372505188, "acc": 0.62890625, "length": 1.0, "mode": "test", "epoch": 36}
{"loss": 0.6554770469665527, "acc": 0.6293324828147888, "length": 1.0000762939453125, "mode": "train", "epoch": 37}
{"loss": 0.6619874238967896, "acc": 0.62890625, "length": 1.0, "mode": "test", "epoch": 37}
{"loss": 0.6556261777877808, "acc": 0.6292154788970947, "length": 1.0001261234283447, "mode": "train", "epoch": 38}
{"loss": 0.6616770029067993, "acc": 0.62890625, "length": 1.0, "mode": "test", "epoch": 38}
{"loss": 0.6551288366317749, "acc": 0.6290976405143738, "length": 1.0006113052368164, "mode": "train", "epoch": 39}
{"loss": 0.6602517366409302, "acc": 0.6287109851837158, "length": 1.0, "mode": "test", "epoch": 39}
{"loss": 0.6550015807151794, "acc": 0.6292153596878052, "length": 1.0000971555709839, "mode": "train", "epoch": 40}
{"loss": 0.6608811020851135, "acc": 0.6291015148162842, "length": 1.0, "mode": "test", "epoch": 40}
{"loss": 0.6551561951637268, "acc": 0.6289225220680237, "length": 1.0001109838485718, "mode": "train", "epoch": 41}
{"loss": 0.6623514890670776, "acc": 0.62890625, "length": 1.0, "mode": "test", "epoch": 41}
{"loss": 0.6548329591751099, "acc": 0.6293326616287231, "length": 1.0006853342056274, "mode": "train", "epoch": 42}
{"loss": 0.6616573333740234, "acc": 0.6287109851837158, "length": 1.0, "mode": "test", "epoch": 42}
{"loss": 0.6546603441238403, "acc": 0.6292154788970947, "length": 1.0000194311141968, "mode": "train", "epoch": 43}
{"loss": 0.6609808206558228, "acc": 0.6287109851837158, "length": 1.0, "mode": "test", "epoch": 43}
{"loss": 0.6550692319869995, "acc": 0.6290983557701111, "length": 1.0000417232513428, "mode": "train", "epoch": 44}
{"loss": 0.6617655158042908, "acc": 0.6287109851837158, "length": 1.0, "mode": "test", "epoch": 44}
{"loss": 0.6552539467811584, "acc": 0.6294512152671814, "length": 1.0010091066360474, "mode": "train", "epoch": 45}
{"loss": 0.6628084182739258, "acc": 0.6285156607627869, "length": 1.0, "mode": "test", "epoch": 45}
{"loss": 0.6547107100486755, "acc": 0.6290980577468872, "length": 1.000037431716919, "mode": "train", "epoch": 46}
{"loss": 0.6608281135559082, "acc": 0.62890625, "length": 1.0, "mode": "test", "epoch": 46}
{"loss": 0.6553378701210022, "acc": 0.6291568279266357, "length": 1.0000461339950562, "mode": "train", "epoch": 47}
{"loss": 0.6610504388809204, "acc": 0.6283202767372131, "length": 1.0, "mode": "test", "epoch": 47}
{"loss": 0.6550394892692566, "acc": 0.6294496059417725, "length": 1.0000050067901611, "mode": "train", "epoch": 48}
{"loss": 0.6630688905715942, "acc": 0.6273437142372131, "length": 1.0, "mode": "test", "epoch": 48}
{"loss": 0.6551700830459595, "acc": 0.6289806365966797, "length": 1.0002379417419434, "mode": "train", "epoch": 49}
{"loss": 0.6616877317428589, "acc": 0.6275390982627869, "length": 1.0, "mode": "test", "epoch": 49}
{"loss": 0.6550058126449585, "acc": 0.6290398836135864, "length": 1.0001280307769775, "mode": "train", "epoch": 50}
{"loss": 0.6622100472450256, "acc": 0.6279296875, "length": 1.0, "mode": "test", "epoch": 50}
{"loss": 0.6556668877601624, "acc": 0.6293324828147888, "length": 1.0000438690185547, "mode": "train", "epoch": 51}
{"loss": 0.6641674041748047, "acc": 0.6273437142372131, "length": 1.0, "mode": "test", "epoch": 51}
{"loss": 0.6571483612060547, "acc": 0.6289228200912476, "length": 1.0000041723251343, "mode": "train", "epoch": 52}
{"loss": 0.6672256588935852, "acc": 0.6175780296325684, "length": 1.0, "mode": "test", "epoch": 52}
{"loss": 0.6562789082527161, "acc": 0.629332423210144, "length": 1.0000605583190918, "mode": "train", "epoch": 53}
{"loss": 0.6633936166763306, "acc": 0.6259765625, "length": 1.0, "mode": "test", "epoch": 53}
{"loss": 0.6555657386779785, "acc": 0.6291559934616089, "length": 1.0001282691955566, "mode": "train", "epoch": 54}
{"loss": 0.6621873378753662, "acc": 0.6273437142372131, "length": 1.0, "mode": "test", "epoch": 54}
{"loss": 0.6545293927192688, "acc": 0.6291568875312805, "length": 1.0000053644180298, "mode": "train", "epoch": 55}
{"loss": 0.6623103618621826, "acc": 0.6283202767372131, "length": 1.0, "mode": "test", "epoch": 55}
{"loss": 0.6551745533943176, "acc": 0.6292736530303955, "length": 1.0000646114349365, "mode": "train", "epoch": 56}
{"loss": 0.6639016270637512, "acc": 0.6285156607627869, "length": 1.0, "mode": "test", "epoch": 56}
{"loss": 0.6551106572151184, "acc": 0.6287471652030945, "length": 1.0000461339950562, "mode": "train", "epoch": 57}
{"loss": 0.6617562770843506, "acc": 0.6281249523162842, "length": 1.0, "mode": "test", "epoch": 57}
{"loss": 0.6548343896865845, "acc": 0.6293326616287231, "length": 1.000048279762268, "mode": "train", "epoch": 58}
{"loss": 0.6622194051742554, "acc": 0.6287109851837158, "length": 1.0, "mode": "test", "epoch": 58}
{"loss": 0.6555728912353516, "acc": 0.6285058856010437, "length": 1.0007853507995605, "mode": "train", "epoch": 59}
{"loss": 0.6641984581947327, "acc": 0.62890625, "length": 1.0, "mode": "test", "epoch": 59}
{"loss": 0.6553546786308289, "acc": 0.6292145252227783, "length": 1.0006757974624634, "mode": "train", "epoch": 60}
{"loss": 0.6645578145980835, "acc": 0.6285156607627869, "length": 1.0, "mode": "test", "epoch": 60}
test accuracy: 0.6309999227523804
